docs/source/changelog.rst

```diff
@@ -6,13 +6,27 @@ Changelog
     * Enhancements
     * Fixes
     * Changes
+        * Remove the ability to stack transform primitives (:pr:`1119`)
+        * Sort primitives passed to ``dfs`` to get consistent ordering of features\* (:pr:`1119`)
     * Documentation Changes
     * Testing Changes
         * Better test case for normalizing from no time index to time index (:pr:`1113`)
 
+    \*When passing multiple instances of a primitive built with ``make_<trans/agg>_primitive``, 
```

Comment:
Is this what we want?

![image](https://user-images.githubusercontent.com/64278226/91594363-cc630e80-e92f-11ea-88d5-14479908f39e.png)

Reply:
Few last thoughts about the changelog
* I think for the footnote, leaving a space between the asterisk and the first word is common "* When ..."
* I think writing out `make_trans_primitive` and `make_agg_primitive` separately will be easier to read/comprehend.  This is a footnote so we're not space limited.
* I think we can simplify the end of this note to be "to ensure the same order of features" instead of mentioning runs